app/author_parse_logic.py

```diff
@@ -0,0 +1,87 @@
+import csv
+import os
+
+from datetime import datetime
+
+from bs4 import BeautifulSoup
+
+from app.description import Author
+from app.description import CLASS_QUOTE, HTML_PARSER
+
+import requests
+
+from app.description import BASE_URL, CLASS_AUTHOR
+
+
+def write_author_to_file(file_name: str, author: Author) -> None:
+    file_exists = os.path.exists(file_name)
+
+    with open(file_name, "a+", newline="", encoding="utf-8") as author_file:
+        field_names = [
+            "name",
+            "born_date",
+            "born_city",
+            "born_country",
+            "description"
+        ]
+
+        author_writer = csv.DictWriter(author_file, fieldnames=field_names)
+
+        if not file_exists:
+            author_writer.writeheader()
+
+        author_writer.writerow({
+            "name": author.name,
+            "born_date": author.born_date,
+            "born_city": author.born_city,
+            "born_country": author.born_country,
+            "description": author.description
+        })
+
+
+def insert_author_to_dict(author_soup: BeautifulSoup) -> Author:
+    return Author(
+        name=author_soup.select_one(".author-title").text,
+        born_date=datetime.strptime(
+            author_soup.select_one(".author-born-date").text, "%B %d, %Y"
+        ),
+        born_city=author_soup.select_one(
+            ".author-born-location").text.split()[1],
+        born_country=author_soup.select_one(
+            ".author-born-location").text.split()[2],
+        description=author_soup.select_one(
+            ".author-description").text,
+    )
+
+
+def get_authors_from_page(soup: BeautifulSoup) -> dict:

```

Comment:
Why do attached to authors so much? Your task is named "Scrape quotes"
You just have anything with author prefix